src/h5web/App.test.tsx

```diff
@@ -126,9 +126,115 @@ test('navigate groups in explorer', async () => {
 
   expect(queryByRole('treeitem', { name: '1_cormap' })).not.toBeInTheDocument();
 
-  // `waitFor` avoids warning due to async work occuring after completion of test
+  // `waitFor` avoids warning due to async work occurring after completion of test
   await waitFor(() => {
     expect(groupBtn).toHaveAttribute('aria-selected', 'true');
     expect(groupBtn).toHaveAttribute('aria-expanded', 'false');
   });
 });
+
+test('select a scalar dataset and display the value', async () => {
```

Comment:
I was trying to write these from the point of view of the user -- trying to identify their purpose when using the app -- so `visualize scalar dataset` would make more sense, I think.